includes/class-cdn-purger.php

```
@@ -0,0 +1,153 @@
+
+			$service = isset( $cache['cdnPurgeService'] ) ? sanitize_text_field( (string) $cache['cdnPurgeService'] ) : 'none';
+
+			if ( 'cloudflare' === $service ) {
+				return self::purge_cloudflare( $cache );
+			}
+			if ( 'varnish' === $service ) {
+				return self::purge_varnish( $cache );
+			}
+
+			return true;
+		}
```

PHPMD (2.15.0)

[error] The method purge_all is not named in camelCase. (undefined)

(CamelCaseMethodName)